### PR TITLE
PropertyAddress API

### PR DESCRIPTION
**Pull request by aleh-rudzko**
added property address API, swagger, service, models, tests

### CODE REVIEW

**Comment by aleh-rudzko** [comment]
@Evilart86 pls, review this after merge https://github.com/aleh-rudzko/mars/pull/11

File: src/server/interfaces/propertyAddress.ts [hunk 1]
```diff
@@ -0,0 +1,8 @@
+export interface PropertyAddress {
```
Comment (Evilart86) [inline src/server/interfaces/propertyAddress.ts:1]: Rename PropertyAddress to AddressProperty
I Think that "Property" is more common name than "Address". There may be many properties with a specific implementation for each. For example NameProperty, AddressProperty, PhoneProperty e.t.c.

Comment (aleh-rudzko) [reply src/server/interfaces/propertyAddress.ts:1]: @Evilart86 Property is physical office or some building.

Comment (Evilart86) [reply src/server/interfaces/propertyAddress.ts:1]: okey i understand

**Pull request merged**